pkg/Library/Designer/Designer.js

```diff
@@ -184,15 +182,18 @@ onNodeDelete({eventlet: {key}, graph}, state) {
   };
 },
 
-onNodePosition({eventlet: {key, value}, graph, layout}, state) {
+onNodePosition({eventlet: {key, value}, graph, layoutId}, state) {
   const node = this.findNodeByParticle(key, graph, state.recipes);
   if (!node) {
     return this.selectNode(null, state);
   }
+  graph.layout ??= {};
```

Comment:
I believe these three lines are equivalent to:

`((graph.layout ??= {})[layoutId] ?? = {})[node.id] = value;`

Reply:
great, thanks! done